content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -191,6 +191,30 @@ details are abstracted away. That abstraction and separation of concerns simplif
 system semantics, and makes it feasible to extend the cluster's behavior without
 changing existing code.
 
+## Pod update and replacement
+
+As mentioned in the previous section, when the Pod template for a workload
+resource is changed, the controller will create new Pods based on the updated
+template instead of updating or patching the existing Pods.
+
+Kubernetes doesn't prevent you from managing Pods directly. It is possible to
+update a Pod in place. However, the Pod update operation has some limitations:
```

Comment:
nit: Does it make sense to replace "in place" with: 
It is possible to update a running Pod.

nit: What exactly is the Pod "update operation"?
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#patch-pod-v1-core

Reply:
update include 'patch' and 'replace', revised.